docs/en/status-communities/create-a-status-community.md

```diff
@@ -55,7 +55,7 @@ After you create a community, you can customize its settings at any time.
 === "Desktop"
 
     1. From the navigation sidebar, click your community.
-    1. On top of the channel sidebar, click your community logo.
+    1. On top of the channels sidebar, click your community logo.
```

Comment:
It should actually be 'channel sidebar' :) 'Channel' is a noun that acts as an adjective modifying 'sidebar', so it should be singular